modules/aws/account.go

```diff
@@ -3,14 +3,15 @@ package aws
 import (
 	"errors"
 	"strings"
-	"testing"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/sts"
+
+	_ "github.com/gruntwork-io/terratest/modules/testing"
```

Comment:
Why import with the `_`?

Reply:
so as not to qualify it with `testing.TestingT` (which may be confusing to others on the provenance of the interface)
I can change the pkg name on import, would that be more acceptable ?

Reply:
> which may be confusing to others on the provenance of the interface

Confusing in what sense?